### PR TITLE
refac: Use latest Messaging

### PR DESCRIPTION
**Pull request by dstenroejl**
## Description

Refactor code to use types implemented in Messaging.

## References

https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-mosaic/293

## Checklist
- [ ] Should the change be behind a feature flag?
- [ ] Can the feature be meaningfully disabled or circumvented if there are issues (e.g., database-breaking changes)?
- [ ] Has it been considered whether data is being delivered to the wrong actor?
- [x] Subsystem test executed (dev_002/dev_003):
  - `dev_002`: https://github.com/Energinet-DataHub/dh3-environments/actions/runs/10830194314
- [ ] Is there time to monitor state of the release to Production?
- [x] Reference to the task

### CODE REVIEW

File: source/B2BApi/B2BApi.csproj [hunk 1]
```diff
@@ -35,7 +35,7 @@ limitations under the License.
         <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Timer" Version="4.3.1" />
         <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.17.4" OutputItemType="Analyzer" />
         <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.23.0" />
-        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.1.0"/>
+        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.2.0-alpha-140" />
```
Comment (dstenroejl) [inline source/B2BApi/B2BApi.csproj:38]: Use release

File: source/BuildingBlocks.Application/BuildingBlocks.Application.csproj [hunk 2]
```diff
@@ -12,7 +12,7 @@
       <PackageReference Include="AspNetCore.HealthChecks.AzureStorage" Version="7.0.0" />
       <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.18.1" />
       <PackageReference Include="Energinet.DataHub.Core.App.Common" Version="13.1.0" />
-        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.1.0"/>
+        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.2.0-alpha-140" />
```
Comment (dstenroejl) [inline source/BuildingBlocks.Application/BuildingBlocks.Application.csproj:15]: Use release

File: source/IntegrationEvents.Application/IntegrationEvents.Application.csproj [hunk 3]
```diff
@@ -14,7 +14,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.1.0"/>
+        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.2.0-alpha-140" />
```
Comment (dstenroejl) [inline source/IntegrationEvents.Application/IntegrationEvents.Application.csproj:17]: Use release

File: source/IntegrationEvents.Infrastructure/IntegrationEvents.Infrastructure.csproj [hunk 4]
```diff
@@ -16,7 +16,7 @@
     <ItemGroup>
       <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.13.5" />
       <PackageReference Include="Dapper" Version="2.1.35" />
-        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.1.0"/>
+        <PackageReference Include="Energinet.DataHub.Core.Messaging" Version="5.2.0-alpha-140" />
```
Comment (dstenroejl) [inline source/IntegrationEvents.Infrastructure/IntegrationEvents.Infrastructure.csproj:19]: Use release

**Review by RasmusGraabaek — DISMISSED** [review]

**Review by RasmusGraabaek — APPROVED** [review]

**Pull request merged**